fms_core/admin.py

```diff
@@ -196,13 +207,14 @@ class IndividualAdmin(AggregatedAdmin):
 
 
 @admin.register(ContainerMove)
-class ContainerMoveAdmin(ImportMixin, admin.ModelAdmin):
+class ContainerMoveAdmin(CustomImportMixin, admin.ModelAdmin):
     resource_class = ContainerMoveResource
     actions = None
     list_display_links = None
 
     def changelist_view(self, request, extra_context=None):
         extra_context = {"title": "Move Containers"}
+        extra_context['submission_template'] = (static("submission_templates/Container_move.xlsx"))
```

Comment:
theres a few examples of redundant parentheses around these static calls I think but otherwise this looks good!